### DISCUSSION TITLE
How can i zoom around mouse?

### DISCUSSION THREAD

**Discussion by Mt-Youya** [konvajs/konva#1648] [Q&A] [answered]
[codesandbox](https://codesandbox.io/s/affectionate-resonance-2f7lhn?file=/src/pages/home.js)

This image is in the cener of this Layer  and zooming around mouse position

**Comment by lavrton** [comment, marked as answer]
Did you look here https://konvajs.org/docs/sandbox/Zooming_Relative_To_Pointer.html#sidebar ?

**Reply by Mt-Youya** [reply]
I'm so sorry!  I looked at first time when i browsing briefly Konva docs. But i forget it on coding.
